bc_obps/registration/fixtures/mock/operation.json

```diff
@@ -12,7 +12,7 @@
       "naics_code": 19,
       "previous_year_attributable_emissions": "10001",
       "swrs_facility_id": 1001,
-      "bcghg_id": 546,
```

Comment:
I just misread the ticket at first and changed the bcghg id instead of the bc corporate id. When I put it back I didn't remember what the original numbers were, and it doesn't matter for the test, so I just left it